src/features/Chat/components/ActiveWindow/index.tsx

```diff
@@ -26,7 +26,12 @@ import MessageList from './MessageList';
 
 const ActiveWindow = () => {
   const { isTablet } = useGetLayoutMode();
-  const chat = useAppSelector(selectActiveChat);
+  const dispatch = useAppDispatch();
+
+  const activeChat = useAppSelector(selectActiveChat);
+  const defaultChat = useAppSelector(selectDefaultChat);
+  const chat = activeChat ?? defaultChat;
+  dispatch(setActiveChat(chat.buddyId));
```

Comment:
Somehow missed this on my first review. 
This means that the dispatch will be done on every rerender?
We should probably introduce an efffect here and have the buddyId as the dependency.

```javascript
React.useEffect(() => {
    dispatch(setActiveChat(chat.buddyId))
},[chat.buddyId])
```

It is more apparent what is happening there then

Reply:
or is this just a one-time effect? In that case just leave the array empty

Reply:
Good observation! Fixed this!